src/unix.rs

```diff
@@ -213,9 +193,8 @@ impl State {
             }
         }
 
-        let mut map = self.map.lock().unwrap();
-        let (_write, ret) = map.remove(&(remove.child as *mut Child)).unwrap();
```

Comment:
Was this even tested? This removed the declaration of `ret` without removing or updating its use below.

Reply:
Apparently it wasn't. I'm so sorry, this is very embarrassing! I submitted #26 to revert this change.

Thank you very, very much for spotting this! CI is no longer running on this repo, and this would have been even *more* embarrassing if it got released.

Reply:
No problem, it happens. We should look at getting CI set up though.